### ISSUE TITLE
react-bootstrap 2.0.1 contains ghost PopoverContent.d.ts

### ISSUE THREAD

**Issue by akx** [react-bootstrap/react-bootstrap#6114]
## Describe the bug

The npm tarball for react-bootstrap 2.0.1 contains react-bootstrap/{cjs,esm}/PopoverContent.d.ts which should not exist since #5677.

This causes TypeScript errors in certain configurations:

```
TS2305: Module '"./helpers"' has no exported member 'BsPrefixPropsWithChildren'.
```

## To Reproduce

1. Examine the contents for the 2.0.1 tarball:
  ```
  $ npm pack react-bootstrap 2>&1 | grep PopoverContent
  npm notice 316B    cjs/PopoverContent.d.ts
  npm notice 316B    esm/PopoverContent.d.ts
  ```

## Reproducible Example

See above.

## Expected behavior

No ghost files.

## Environment (please complete the following information)

- React-Bootstrap Version: 2.0.1

## Additional context

This sounds like the 2.0.1 tarball was built from a source tree that had not been properly `git clean`ed before build and package.

Maybe the release process should be done in a guaranteed isolated environment such as GitHub Actions.

**Comment by kyletsang** [comment]
Sorry my bad.  I re-released under 2.0.2.  Let me know if you run into issues.

**Comment by akx** [comment]
All good, @kyletsang. Thanks for the amazing response time!
